cobrawap/pipeline/utils/Snakefile

```diff
@@ -150,4 +150,4 @@ rule template_all:
     shell:
         """
         {params.command} "{input.data}" "{output.data}"
-        """
\ No newline at end of file
+        """
```

Comment:
Files should end with a newline.

Reply:
Same as above, it ends with a blank line.